modules/setting/storage.go

```diff
@@ -47,6 +47,7 @@ type MinioStorageConfig struct {
 	InsecureSkipVerify bool   `ini:"MINIO_INSECURE_SKIP_VERIFY"`
 	ChecksumAlgorithm  string `ini:"MINIO_CHECKSUM_ALGORITHM" json:",omitempty"`
 	ServeDirect        bool   `ini:"SERVE_DIRECT"`
+	BucketLookUpType   string `ini:"MINIO_BUCKET_LOOKUP_TYPE"`
```

Comment:
Should this be added below to the `getDefaultStorageSection` to default to `auto`? It may simplify the final branch of the if-check in `NewMinioStorage`

Reply:
Thanks, I added it to getDefaultStorageSection.